ClientApp/viking-vault/src/UserDashboard/BlockedServices.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import {constants} from "../Resources/Constants";
+import SideBar from "../Common/SideBar";
+import TopBar from "../Common/TopBar";
+import UserIcon from "../Common/UserIcon";
+import "../Common/styles.css";
+
+interface IBlockedServicesState {
+    userHasCard: boolean | null;
+    isCardBlocked: boolean | null;
+}
+
+class BlockedServices extends React.Component<any, IBlockedServicesState> {
+    constructor(props: any) {
+        super(props);
+        this.state = {
+            userHasCard: null,
+            isCardBlocked: null
+        }
+    }
+
+    componentDidMount() {
+        this.checkUserHasCard();
+        this.checkCardIsBlocked();
+    }
+
+    checkUserHasCard = () => {
+        const token = sessionStorage.getItem("Authentication-Token");
+        if(token === null) {
+            return;
+        }
+        fetch(constants.baseUrl+"card", {
```

Comment:
You could extract constants.baseUrl+"card" in a constant, and name it accordingly